composer.json

```
@@ -32,19 +32,15 @@
         "access token"
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "ext-json": "*",
         "php-http/discovery": "^1.7",
         "psr/http-client": "^1.0",
         "psr/http-message": "^1.0 || ^2.0",
         "psr/simple-cache": "^1.0 || ^2.0 || ^3.0",
         "spomky-labs/base64url": "^2.0.1",
         "symfony/polyfill-mbstring": "^1.15",
-        "web-token/jwt-checker": "^2.2.0 || ^3.0",
-        "web-token/jwt-core": "^2.2.0 || ^3.0",
-        "web-token/jwt-key-mgmt": "^2.2.0 || ^3.0",
-        "web-token/jwt-signature": "^2.2.0 || ^3.0",
-        "web-token/jwt-signature-algorithm-rsa": "^2.2.0 || ^3.0"
+        "web-token/jwt-library": "^2.2.0 || ^3.0"
```

This doesn't make sense, this package exists only since 3.3: https://packagist.org/packages/web-token/jwt-library
```suggestion
        "web-token/jwt-library": "^3.3"
```